src/Microsoft.Sbom.Api/Config/Validators/ManifestInfoValidator.cs

```diff
@@ -16,20 +17,16 @@ namespace Microsoft.Sbom.Api.Config.Validators;
 public class ManifestInfoValidator : ConfigValidator
 {
     public ManifestInfoValidator(IAssemblyConfig assemblyConfig)
-        : base(typeof(ValidUriAttribute), assemblyConfig)
-    {
-    }
-
-    public ManifestInfoValidator(Type supportedAttribute, IAssemblyConfig assemblyConfig)
-        : base(supportedAttribute, assemblyConfig)
+        : base(typeof(ValidManifestInfoAttribute), assemblyConfig)
     {
     }
 
     public override void ValidateInternal(string paramName, object paramValue, Attribute attribute)
     {
         if (paramValue is not null && paramValue is ManifestInfo manifestInfo && !Constants.SupportedSpdxManifests.Contains(paramValue as ManifestInfo))
```

Comment:
We may be able to simplify this just a bit by using the leveraging the `manifestInfo` variable that is specified:
```suggestion
        if (paramValue is not null && paramValue is ManifestInfo manifestInfo && !Constants.SupportedSpdxManifests.Contains(manifestInfo))
```